docs/General/createCIRAConfig.md

```diff
@@ -43,6 +54,7 @@ A CIRA Configuration provides the Remote Provisioning Client (RPC) the required
 
 Profiles provide configuration information to the AMT Firmware during the activation process with the Remote Provisioning Client (RPC). Profiles also distinguish between activating in: 
 
+**[Admin Control Mode (ACM):](createProfileACM.md)** ACM mode supports the same CCM features in addition to Keyboard, Video, Mouse (KVM) or Redirection **without user consent**. This means it is **not necessary** to have a person on-site to remote in and manage an edge device. In most IoT use cases, edge devices such as digital signage or kiosks may not be easily accessible or have available employees nearby. ACM mode proves immensely helpful in these scenarios.
+
 **[Client Control Mode (CCM):](createProfileCCM.md)** This mode offers nearly all manageability features. However, it does not currently support Keyboard, Video, Mouse Control. For other features, such as Serial-Over-LAN or Storage Redirection (IDE-R, USB-R), **user consent is required**.
```

Comment:
Do we distinguish between what is a limit of AMT vs the Toolkit here? Or have Toolkit take priority and say 'User consent features, such as blah,blah,blah, are not currently supported.  CCM can still provide power control, audit.....' and so on.  Thoughts?

Reply:
I actually began this after the Corey's WT, and I think that was a mistake. I flipped order, thinking it was more important than it is. Let's check this in, flipped back to the way it was, and then regroup with a new checkin. This is huge topic, this idea of a) what goes first b) what should be mentioned and explained c) what we should say. 

My general gut feeling is that this is currently all wrong. We should actually make it much simpler: 

**[Client Control Mode (CCM):](createProfileCCM.md)** This mode supports nearly all manageability features available in Intel AMT. However, it does not currently support Keyboard, Video, Mouse Control. For other features, such as Serial-Over-LAN or Storage Redirection (IDE-R, USB-R), **user consent is required**. This mode enables quick configuration of a Hello World experience as it does not require a provisioning certificate from a Certificate Authority.

[Admin Control Mode (ACM):](createProfileACM.md)** ACM mode offers full Intel AMT support, including Keyboard, Video, Mouse (KVM) or Redirection **without user consent**. **Use** this mode if you've already purchased a provisioning certificate from a Certificate Authority.**

Unsure? Click the links above to see more details about these options.

My thinking is that it is important for people that already have the PC to find the option that they need quickly, while emphasizing the ease-of-use for CCM. I think making the quick decisions makers read a bunch of details first is a bad move.

Reply:
I think that is the right direction, we can resolve this for now and discuss this separately.